modules/lib/maintainers.nix

```diff
@@ -93,4 +93,10 @@
     githubId = 34393802;
     name = "Malte Voos";
   };
+  kranzes = {
+    email = "personal@ilanjoselevich.com";
+    github = "Kranzes";
+    githubId = 56614642;
+    name = "Ilan Joselevich";
+  };
```

Comment:
Are you a maintainer in NixOS' `maintainers-list.nix`?
If so, you don't have to add yourself here.

Otherwise you will need to update `meta.maintainers` to `meta.maintainers = [ lib.hm.kranzes ]`.

Reply:
I am not, guess i will use lib.hm, Thanks!!!